### DISCUSSION TITLE
Having trouble converting from `org` to `man`

### DISCUSSION THREAD

**Discussion by CSUG-Cole** [jgm/pandoc#9631] [Q&A] [answered]
Maybe I am unfamiliar with `org` syntax, but I'm unable to reproduce the `markdown` to `man` example on the official site. 

Here is a working `markdown` 
```md
% MARKDOWN(1) MARKDOWN User Manual
% Me
% April 3, 2024

# OPTIONS 
**-h, --help**
:    test
```
to `man` example.
```man
.\" Automatically generated by Pandoc 3.1.8
.\"
.TH "MARKDOWN" "1" "April 3, 2024" "MARKDOWN User Manual" ""
.SH OPTIONS
.TP
\f[B]-h, \[en]help\f[R]
test
.SH AUTHORS
Me.
```

And here is an `org` document with various attempts to achieve the same thing
```org
#+title: ORG(1)

* OPTIONS
*-h, --help* ::    test

*-v, --version*
0.0.1

*-t, --test*

hmm...

*-d, --debug*
: ??
```
and the `man` page it produces.
```man
.\" Automatically generated by Pandoc 3.1.8
.\"
.TH "ORG" "1" "" "" ""
.SH OPTIONS
\f[B]-h, \[en]help\f[R] :: test
.PP
\f[B]-v, \[en]version\f[R] 0.0.1
.PP
\f[B]-t, \[en]test\f[R]
.PP
hmm\&...
.PP
\f[B]-d, \[en]debug\f[R]
.IP
.EX
??
.EE
```

Thank you for any help.

**Comment by jgm** [comment, marked as answer]
Org mode definition list syntax is:
```
- *-h, --help* ::    test
```
You are missing the initial `-`.  Also missing the `::` on subsequent examples.

**Reply by CSUG-Cole** [reply]
Perfect, thank you for the help!

**Reply by CSUG-Cole** [reply]
I just realized that I could convert from man to org as well. This provided a complete example of how one converts to the other. Is that mentioned anywhere in the documents? If not, I think it would be worth mentioning somewhere on maybe the examples page. It is impractical to provide an example of every conversion, but this is something any user can do given a document of the desired output, where they are unfamiliar with the syntax of the source or how exactly one gets converted to another. Just a thought. Thanks again for the help and for maintaining this great project!
